src/Bicep.Cli/CommandLine/ArgumentParser.cs

```diff
@@ -17,21 +17,16 @@ public static class ArgumentParser
             }
 
             // parse verb
-            switch (args[0].ToLowerInvariant())
+            return (args[0].ToLowerInvariant()) switch
```

Comment:
Do you know why this PR is showing my changes as included? My guess is you merged with a squash, or merged directly from my branch rather than `origin/main`. It should be possible to sort out with the following, though you may get some conflicts to fix:

```sh
git fetch
git merge --no-ff origin/main
... handle conflicts, commit merge ...
git push
```